tests/conftest.py

```diff
@@ -177,6 +177,14 @@ def docker_env_vars(versionfile, monkeypatch):
     monkeypatch.setenv("MAIL_SERVER", "localhost")
     monkeypatch.delenv("MAIL_SUPPRESS_SEND", raising=False)
 
+    treestatus_app = (
+        "1"
+        if hasattr(request.module, "TREESTATUS_APP")
+        and request.module.TREESTATUS_APP is True
+        else ""
+    )
+    monkeypatch.setenv("TREESTATUS_APP", treestatus_app)
+
```

Comment:
This is still a little awkward, but I don't think there's a way to completely remove this awkwardness with the current implementation taking the environment variable approach. Ideally things can be done cleanly in test fixtures with the behaviour unchanged from module to module. Probably the only way to do this cleanly here would be to rewrite the `app` fixture in `test_treestatus`.

However given that some of this functionality will be ported/reimplemented in django-lando, an interim clean up here would suffice -- I would just remove the `test_treestatus.TREESTATUS_APP` variable altogether and just check the module name.


```suggestion
    if request.module.__name__ == "tests.test_treestatus":
        monkeypatch.setenv("TREESTATUS_APP", treestatus_app)
```